conda/rapids-gpu-bdb.yml

```diff
@@ -4,8 +4,8 @@ channels:
   - conda-forge
```

Comment:
Also worth moving Conda-forge above nvidia as the priority order changed here.

Reply:
can fix in a followup